nautobot/dcim/forms.py

```diff
@@ -2359,6 +2366,59 @@ class DeviceFilterForm(
     tags = TagFilterField(model)
 
 
+class DeviceAddToClustersForm(BootstrapMixin, forms.Form):
+    """Form for adding a device to one or more clusters."""
+
+    location = DynamicModelChoiceField(
+        queryset=Location.objects.all(),
+        required=False,
```

Comment:
Possibly filter this field by `query_params={"content_type": ["virtualization.cluster"]}`? Not 100% certain if that makes sense for the use case.